web_widget_table/static/js/web_widget_table.js

```diff
@@ -0,0 +1,55 @@
+/* License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_widget_table', function (require) {
+    "use strict";
+
+    var registry = require('web.field_registry');
+    var AbstractField = require('web.AbstractField');
+    var FieldDataTable = AbstractField.extend({
+        className: 'oe_form_field_table',
+        supportedFieldTypes: ['char'],
+        init: function init () {
```

Comment:
:fire: Remove this method, it's doing nothing.

Reply:
Done